docs/5_3_scheduled_execution_example.adoc

```diff
@@ -57,14 +57,18 @@ In the context of the task being executed every 10 ms, the FMU initiates the sch
 include::examples/snippets.c[tags=SE_fmu_activateMP10ms]
 ----
 
-In <<fmi3GetIntervalDecimal>> the Clock's interval qualifier is reset to <<fmi3GetIntervalDecimal, `fmi3IntervalUnchanged`>>.
+In <<fmi3GetIntervalDecimal>> the clock's interval qualifier is reset to <<fmi3GetIntervalDecimal, `fmi3IntervalUnchanged`>>.
+To ensure consistency it may be necessary to prevent preemption.
```

Comment:
This is a bit vague. It's unclear whether we are forcing this, or just recommending it.

Reply:
... a bit vague and still not clear, who must prevent preemption: importer of FMU?

Reply:
> "Clock" should be upper case if it refers to Clocks in FMI (as opposed to a wall clock e.g.).

@andreas-junghanns What do you think? ... the term occurs 383 times in the document.

Reply:
> This is a bit vague

I changed the text to explicitly name the FMU.